packages/app/pages/index.tsx

```diff
@@ -37,6 +37,36 @@ export default function Home({ clubs }: HomeProps) {
       refreshData();
     });
   }, []);
+
+  // web push code
+  const check = () => {
+    if (!("serviceWorker" in navigator)) {
+      throw new Error("No Service Worker support!");
+    }
+    if (!("PushManager" in window)) {
+      throw new Error("No Push API Support!");
+    }
+  };
+
+  const requestNotificationPermission = async () => {
+    if (Notification.permission === "granted") {
+      console.log(`permission previously granted`);
+    } else if (Notification.permission === "denied") {
+      console.log("permission previously denied");
+      const permission = await window.Notification.requestPermission();
```

Comment:
this doesn't do anything, because the API is disabled if permission is denied. we should remove it
```suggestion
```